app/controllers/icon_redirects_controller.rb

```diff
@@ -1,4 +1,9 @@
 class IconRedirectsController < ApplicationController
+
+  before_action {
+    expires_in(1.day, public: true)
+  }
```

Comment:
```suggestion
  before_action { expires_in(1.day, public: true) }
```

Minor nit pick, https://rubystyle.guide/#single-line-blocks

Reply:
Not having rubocop in the main rake task bites me every time.